notary-server/Dockerfile

```diff
@@ -9,7 +9,7 @@ WORKDIR ${INSTALLDIR}
 
 RUN set -eux; \
     apk add --no-cache --virtual build-deps git go make musl-dev; \
-    go version | grep 'go1.12.10 '; \
+    go version | grep 'go1.12.12 '; \
```

Comment:
What do you think about adjusting this to be something like `go version | grep -F 'go 1.12.'` instead?

(Also, do you think we're safe to upgrade from Alpine 3.10 to 3.12 and thus update Go, or do we need to stick to Alpine 3.10 and/or Go 1.12?)